qiita_pet/handlers/api_proxy/tests/test_prep_template.py

```diff
@@ -74,8 +74,9 @@ def test_prep_template_ajax_get_req(self):
                'name': "Prep information 1",
                'files': ["uploaded_file.txt"],
                'download_prep_id': 24,
-               'download_qiime_id': 21,
-               'other_filepaths': ['1_prep_1_19700101-000000.txt',
+               'other_filepaths': ['1_prep_1_qiime_19700101-000000.txt',
```

Comment:
This means that QIIME mapping files will be stored, but not indexed in the main site, right?

Reply:
It means that they will be displayed in the list of old files, like here:
![Screen Shot 2020-11-17 at 12 27 25 PM](https://user-images.githubusercontent.com/2014559/99437920-520fae00-28d0-11eb-87a4-f0e2d12bfe0f.png)

Reply:
Got it, thanks!